caip-browser_extension_provider_communication.md

```diff
@@ -0,0 +1,141 @@
+---
+# Every document starts with a front matter in YAML enclosed by triple dashes.
+# See https://jekyllrb.com/docs/front-matter/ to learn more about this concept.
+caip: <to be assigned>
+title: Web extension provider communication
+author: Mark Stacey (@Gudahtt), Jiexi Luan (@jiexi)
+discussions-to: <URL>
+status: Draft
+type: Standard
+created: 2022-11-28
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the CAIP.-->
+The proposal aims to standardize how providers communicate with web extensions, making the process more efficient and secure.
```

Comment:
I wonder if we should be a bit more specific about what provider means in this context?

Reply:
or perhaps this context can be taken for granted given this is landing in CASA?

Reply:
we can totally elaborate on what a provider is. You asking this question has me thinking that we should be only focus on providers in so much as they provide a convenience wrapper for wallet communication. That is to say, we should more broadly say "Dapp" in place of "provider" in several places in this document

Reply:
Yeah this is tricky. Provider is a potentially confusing word since it operates as this nebulous thing that "provides" an interface for blockchains and here we're changing our assumption of where this provider lives - previously injected, now BYOP - in the stack without even providing (:wink:) that context.

Reply:
how about this https://github.com/jiexi/CAIPs/pull/1/commits/4205dc5646d82c65c1901a821f240a09d0d2f83c?

Reply:
adding this comment I left on the commit here:

> 🤔 it wouldn't be injected anymore though?